corehq/apps/data_interfaces/views.py

```diff
@@ -265,6 +267,10 @@ class CaseGroupCaseManagementView(DataInterfaceSection, CRUDPaginatedViewMixin):
     deleted_items_header = ugettext_noop("Removed Cases:")
     new_items_header = ugettext_noop("Added Cases:")
 
+    @method_decorator(require_permission(Permissions.edit_messaging))
```

Comment:
nit: `@method_decorator` can go on the class instead and avoid needing to override the `dispatch` method:
```python
@method_decorator(require_permission(Permissions.edit_messaging), name="dispatch")
```
Saves a bit of boilerplate, but it's otherwise the same.

Reply:
super fair I like this better as well